CHANGELOG.md

```diff
@@ -11,6 +11,8 @@ For details about compatibility between different releases, see the **Commitment
 
 ### Added
 
+- Redis rate limiting store.
+
```

Comment:
Please change to `Option to store rate limits in Redis. When used, the rate limits are applied over the entire cluster instead of per-instance.`